src/s3/explorer/s3FolderNode.ts

```diff
@@ -36,7 +36,7 @@ export class S3FolderNode extends AWSTreeNodeBase implements AWSResourceNode, Lo
     public constructor(
         public readonly bucket: Bucket,
         public readonly folder: Folder,
-        private readonly s3: S3Client,
+        public readonly s3: S3Client,
```

Comment:
Change this back to private or is it still needed as public?

Reply:
I think I may need it public, depending on what the discussion below comes down to